<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>581</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>QListWidget</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>当前项变化</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>可编辑</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>新建项目</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/Lenovo/Desktop/images/check2.ico</normaloff>C:/Users/Lenovo/Desktop/images/check2.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新建项目1</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>C:/Users/Lenovo/Desktop/images/check2.ico</normaloff>C:/Users/Lenovo/Desktop/images/check2.ico</iconset>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>571</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QToolButton" name="tBtnSelectItem">
         <property name="text">
          <string>tBtnSelectItem</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tBtnSelAll">
         <property name="text">
          <string>tBtnSelALL</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tBtnSelNone">
         <property name="text">
          <string>tBtnSelNone</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="tBtnSelInvs">
         <property name="text">
          <string>tBtnSelInvs</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>QTableWidget</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>QTreeWidget</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>190</width>
      <height>503</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="lineWidth">
     <number>-2</number>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>190</width>
       <height>398</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>C:/Users/Lenovo/Desktop/images/410.bmp</normaloff>C:/Users/Lenovo/Desktop/images/410.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QListwidget操作</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QToolButton" name="tBtnListIni_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>tBtnListIni</string>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="tBtnListAppend_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>tBtnListAppend</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QToolButton" name="tBtnListDelete_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>tBtnListDelete</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="tBtnListInsert_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>tBtnListInsert</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="tBtnListClear_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>tBtnListClear</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>190</width>
       <height>398</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>C:/Users/Lenovo/Desktop/images/408.bmp</normaloff>C:/Users/Lenovo/Desktop/images/408.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QTreewidget操作</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>191</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>190</width>
       <height>398</height>
      </rect>
     </property>
     <attribute name="icon">
      <iconset>
       <normaloff>C:/Users/Lenovo/Desktop/images/412.bmp</normaloff>C:/Users/Lenovo/Desktop/images/412.bmp</iconset>
     </attribute>
     <attribute name="label">
      <string>QTablewidget操作</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListAppend"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListIni"/>
   <addaction name="actListDelete"/>
  </widget>
  <action name="actListAppend">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/128.bmp</normaloff>C:/Users/Lenovo/Desktop/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/delete1.bmp</normaloff>C:/Users/Lenovo/Desktop/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/424.bmp</normaloff>C:/Users/Lenovo/Desktop/images/424.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="actListIni">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/316.bmp</normaloff>C:/Users/Lenovo/Desktop/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/324.bmp</normaloff>C:/Users/Lenovo/Desktop/images/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除项</string>
   </property>
   <property name="toolTip">
    <string>删除项</string>
   </property>
  </action>
  <action name="actSelALL">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/exit.bmp</normaloff>C:/Users/Lenovo/Desktop/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/Desktop/images/124.bmp</normaloff>C:/Users/Lenovo/Desktop/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actSelPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actSelInvs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
